unison-src/transcripts-using-base/fix2027.output.md

```diff
@@ -53,7 +53,6 @@ myServer = unsafeRun! '(hello "127.0.0.1" "0")
   change:
   
     ⊡ Previously added definitions will be ignored: Exception
-      Exception.raise
```

Comment:
We chatted about this and decided there's not much of a reason to print out duplicate constructors when the type itself is already listed.

Adding Constructors to the Slurp Components explicitly gives us a lot more flexibility in the future if we decide to change what we show.